src/test/resources/StandaloneConfigTest/invalidAuthorAliases_config.json

```diff
@@ -0,0 +1,29 @@
+{
+	"ignoreGlobList": ["collated**"],
+	"authors":
+	[
+		{
+			"githubId": "lithiumlkid",
+			"displayName": "Ahm",
+			"authorNames": ["Ahmad Syafiq"],
+			"ignoreGlobList": ["*.aa1", "**.aa2", "**.java"]
+		},
+		{
+			"githubId": "codeeong",
+			"displayName": "Cod",
+			"authorNames": ["Codee; ls"],
+			"ignoreGlobList": ["**[!(.md)]"]
```

Comment:
can each `_configjson` test file contain just that 1 author that has the invalid character, so its easier to tell which is invalid?

Reply:
in this file, only this line is invalid.
`"authorNames": ["Codee; ls"],`

Reply:
Yea, but as they are other 'valid' `authorName`s here, it may be confusing for someone new to figure out which `authorName` for this file is invalid, hence I suggested should have just 1 author for each file.

Reply:
Hmm, the thing is, it is not very accurate if I only have a single author. 
E.g. my regex can be checking for valid authors and not checking for invalid authors.
I will rename the file for it to be more intuitive.